delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/acls/AclsImpl.scala

```diff
@@ -31,11 +32,19 @@ final class AclsImpl private (
     index: AclsCache
 ) extends Acls {
 
+  private val minimum: Set[Permission] = permissions.minimum
```

Comment:
This can change between being assigned and being used.

Reply:
I provided the same logic as in [1.4.x](https://github.com/BlueBrain/nexus/blob/v1.4.x/delta/src/main/scala/ch/epfl/bluebrain/nexus/iam/acls/package.scala#L50) which I think it is the correct anyway, since we want immutability. We don't want to fetch the ACLs and get different results from the same revision depending on what we do with the permissions collection